backend/apps/core/management/commands/backup_db.py

```
@@ -0,0 +1,45 @@
+            'pg_dump',
+            '-h', str(db_host),
+            '-p', str(db_port),
+            '-U', str(db_user),
+            '-F', 'c',
+            '-f', backup_file,
+            str(db_name)
+        ]
+        
+        self.stdout.write(self.style.NOTICE(f'Starting backup of database {db_name} to {backup_file}...'))
+        try:
+            subprocess.run(command, env=env, check=True)
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)